__docs__/wonder-blocks-testing/exports.render-hook-static.mdx

```diff
@@ -0,0 +1,25 @@
+import {Meta} from "@storybook/blocks";
+
+<Meta
+    title="Packages / Testing / Utilities / Exports / renderHookStatic()"
+    parameters={{
+        chromatic: {
+            disableSnapshot: true,
+        },
+    }}
```

Comment:
Same comment here (no change required as it works fine right now, but I leave it to you in case you can get rid of this param).

Reply:
I've removed this from all the MDX files.